From 8418929c502a21c8fa614eb3464e2e037aa0e903 Mon Sep 17 00:00:00 2001
From: Calvin Johnson <calvin.johnson@nxp.com>
Date: Fri, 27 Oct 2017 11:20:47 +0530
Subject: [PATCH 323/741] staging: fsl_ppfe/eth: Make phy_ethtool_ksettings_get
 return void

commit 02ad431cd61cb27b1d05b76d074a34973bc57b27 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

Make return value void since function never return meaningful value

Signed-off-by: Calvin Johnson <calvin.johnson@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/staging/fsl_ppfe/pfe_eth.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/staging/fsl_ppfe/pfe_eth.c b/drivers/staging/fsl_ppfe/pfe_eth.c
index d26ab5fbb0f8..35c87a04c225 100644
--- a/drivers/staging/fsl_ppfe/pfe_eth.c
+++ b/drivers/staging/fsl_ppfe/pfe_eth.c
@@ -577,7 +577,9 @@ static int pfe_eth_get_settings(struct net_device *ndev,
 	if (!phydev)
 		return -ENODEV;
 
-	return phy_ethtool_ksettings_get(phydev, cmd);
+	phy_ethtool_ksettings_get(phydev, cmd);
+
+	return 0;
 }
 
 /*
-- 
2.17.1

